src/components/side-panel/layer-panel/vis-config-slider.js

```diff
@@ -50,31 +85,123 @@ export default function VisConfigSliderFactory(RangeSlider) {
     disabled,
     onChange,
     inputTheme
-  }) => (
-    <SidePanelSection disabled={Boolean(disabled)}>
-      {label ? (
-        <PanelLabel>
-          {typeof label === 'string' ? (
-            <FormattedMessage id={label} />
-          ) : typeof label === 'function' ? (
-            <FormattedMessage id={label(config)} />
+  }) => {
+    const [custom, setCustom] = useState(false);
+    const [values, setValues] = useState({
+      value: config.visConfig[property],
```

Comment:
you don't need 3 values here, `customValue` should be either a number or a [number, number] if slider is a range

```
    const [customValue, setCustomValues] = useState({
      value: config.visConfig[property]
    });
```

Reply:
Also if you use `customValue` to temporarily store component state value, and `onChange` will update the actual value in the reducer state, you should update `customValue` when value in the reducer state changes. An `useEffect` will do

```js
const value = config.visConfig[property];
const [customValue, setCustomValues] = useState(value);
useEffect(() => {
   setCustomValues(value);
}, [value]);
```